docs/conf.py

```diff
@@ -52,12 +53,13 @@ def __getattr__(cls, name):
 #source_encoding = 'utf-8-sig'
 master_doc = 'index'
 project = _setup.__project__.title()
-copyright = '2015 %s' % _setup.__author__
+copyright = '2015-{0} {1}'.format(datetime.now().year, _setup.__author__)
```

Comment:
The rest of this file uses %-style string interpolation, so IMHO this should be switched back to that rather than changing it to use `.format`.